tools/src/bin/collect.rs

```diff
@@ -186,29 +184,14 @@ struct QueryOptions {
         help_heading = "Collect Request Parameters (Time Interval)"
     )]
     batch_interval_start: Option<u64>,
+
     /// Duration of the collection batch interval, in seconds
     #[clap(
         long,
         requires = "batch_interval_start",
         help_heading = "Collect Request Parameters (Time Interval)"
     )]
     batch_interval_duration: Option<u64>,
-
-    /// Batch identifier, encoded with base64url
-    #[clap(
-        long,
-        conflicts_with_all = ["batch_interval_start", "batch_interval_duration", "current_batch"],
-        help_heading = "Collect Request Parameters (Leader Selected)",
-    )]
-    batch_id: Option<BatchId>,
-    /// Have the aggregator select a batch that has not yet been collected
-    #[clap(
-        long,
-        action = ArgAction::SetTrue,
-        conflicts_with_all = ["batch_interval_start", "batch_interval_duration", "batch_id"],
-        help_heading = "Collect Request Parameters (Leader Selected)",
-    )]
-    current_batch: bool,
```

Comment:
Hmm, so now without explicit `--batch-interval-{start,duration}`, we automatically assume we're operating on a fixed size task.

I suppose that's fine. Maybe we can think of a better way to clean up this CLI for this, but I don't think its important.

Reply:
Yep, that's correct. I thought about this for a bit & came up with the same conclusion you did. We could introduce something like a `--leader-selected` flag to indicate a leader-selected query, but I'm not fully sure it's necessary.